components/homme/src/share/cxx/utilities/SubviewUtils.hpp

```diff
@@ -26,7 +26,7 @@ subview(ViewType<ScalarType * [DIM1], MemSpace, Properties...> v_in, int ie) {
   assert(ie < v_in.extent_int(0));
   assert(ie >= 0);
   return ViewUnmanaged<ScalarType[DIM1], MemSpace>(
-      &v_in.implementation_map().reference(ie, 0));
+      &v_in.impl_map().reference(ie, 0));
```

Comment:
You changed name of member, so this code is/was never built?

Reply:
I think kokkos currently allows both. The old one would still compile now, but it won't sometime in the future.

Reply:
This code is built. We use it every time we take a subview in HOMMEXX.

Reply:
Right. imlementation_map is deprecated and impl_map is the replacement. Currently both work unless you build Kokkos with deprecated code disallowed.